images/resources/META-INF/services/javax.imageio.spi.ImageReaderSpi

```diff
@@ -1 +1,2 @@
-org.intellij.images.util.imageio.CommonsImagingImageReaderSpi
\ No newline at end of file
+org.intellij.images.util.imageio.CommonsImagingImageReaderSpi
+org.intellij.images.util.imageio.svg.SvgImageReaderSpi
```

Comment:
**note:** The reader is working fine when tested with

```kotlin
ImageIO.read(URI.create("https://jb.gg/badges/official.svg").toURL())
```

However I'm struggling to get it working with Swing's `javax.swing.text.html.ImageView` used in a `JEdtorPane`, e.g.

```kotlin
JEditorPane(
    "text/html",
    "<html><body><img src='https://jb.gg/badges/official.svg'></body></html>"
)
```

The underlying code uses the toolkit, e.g.

```kotlin
Toolkit.getDefaultToolkit().createImage(URI.create("https://jb.gg/badges/official.svg").toURL())
```

Eventually followed by a call to `java.awt.Toolkit#prepareImage`